### PR TITLE
Add Social Media Icons (Issue #1391) 

### PR DESCRIPTION
**Pull request by hd719**
This is a pull request for issue #1391 
- Adding social media icons

### CODE REVIEW

File: .bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_WITHOUT: "production"
```
Comment (tomhughes) [inline .bundle/config]: What is this for?

File: Gemfile.lock [hunk 2]
```diff
@@ -38,24 +38,24 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.0)
+    addressable (2.5.1)
```
Comment (tomhughes) [inline Gemfile.lock]: Please don't update the bundle more than is necessary - just do `bundle update social-share-button` rather than updating the whole thing.

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/*
+ *= require_tree .
+ *= require social-share-button
+ *= require_self
+ */
```
Comment (tomhughes) [inline app/assets/stylesheets/application.css]: What is this for? I don't think we use this do we?

File: db/structure.sql [hunk 4]
```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.6
--- Dumped by pg_dump version 9.5.6
+-- Dumped from database version 9.5.4
+-- Dumped by pg_dump version 9.5.4
```
Comment (tomhughes) [inline db/structure.sql]: Please remove this - you haven't changed the schema at all so there's no need for these changes which just reflect your local environment being different.

File: db/structure.sql [hunk 5]
```diff
@@ -130,7 +130,7 @@ CREATE TYPE user_status_enum AS ENUM (
 
 CREATE FUNCTION maptile_for_point(bigint, bigint, integer) RETURNS integer
     LANGUAGE c STRICT
-    AS '$libdir/libpgosm', 'maptile_for_point';
+    AS '/Users/HD/Projects/Open-Source/openstreetmap-website/db/functions/libpgosm', 'maptile_for_point';
```
Comment (mmd-osm) [inline db/structure.sql]: Why is this pointing to files on your local machine now?

File: db/structure.sql [hunk 6]
```diff
@@ -226,8 +226,8 @@ ALTER SEQUENCE changeset_comments_id_seq OWNED BY changeset_comments.id;
 
 CREATE TABLE changeset_tags (
     changeset_id bigint NOT NULL,
-    k character varying(255) DEFAULT ''::character varying NOT NULL,
-    v character varying(255) DEFAULT ''::character varying NOT NULL
+    k character varying DEFAULT ''::character varying NOT NULL,
```
Comment (mmd-osm) [inline db/structure.sql]: Why are all those DDL changes done for a social media button? Seems quite strange.

File: db/structure.sql (hunk 4 above)
Comment (mmd-osm) [inline db/structure.sql]: (i.e. whole file db/structure.sql should not be part of this pull request)

Comment (tomhughes) [inline db/structure.sql]: Isn't that what I said? Was there a reason you felt the need to repeat my comment?

Comment (mmd-osm) [inline db/structure.sql]: I started adding comments further down this file and noticed a bit too late that your comment was actually pertaining to the whole file. So removed those other comments again and added this small reminder. Sorry for the noise. It's gone again.

Comment (gravitystorm) [inline db/structure.sql]: @mmd-osm thanks for helping review this PR, help is definitely welcome

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (hd719) [inline app/assets/stylesheets/application.css]: Hey @tomhughes! 

I had to create a `app/assets/stylesheets/application.css` file for the `social-share-button` it was part of the requirements. 

https://github.com/huacnlee/social-share-button

**Comment by mmd-osm** [comment]
@hd719 : you need to remove ALL of your changes to db/structure.sql from this pull request, not just the version number. None of it is relevant for the social media button, i.e. the whole file db/structure.sql should not be part of this pull request at all.

File: db/structure.sql (hunk 5 above)
Comment (mmd-osm) [inline db/structure.sql]: That's pointing to your local directory now!!

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (tomhughes) [inline app/assets/stylesheets/application.css]: No, you didn't. I already told you where to put that include...

Seriously, the name of the file doesn't matter in the slightest - it just needs to be included in a CSS file that will be loaded on the relevant page(s).

Comment (tomhughes) [inline app/assets/stylesheets/application.css]: Incidentally as I said I don't think anything will actually load `application.css` so putting it there will achieve nothing.

**Comment by tomhughes** [comment]
Indeed as @mmd-osm all the changes in that file need to be removed - in fact the entire first commit needs to go.

**Comment by hd719** [comment]
@tomhughes and @mmd-osm I see okay, will do. 

Sorry about that!

**Comment by hd719** [comment]
Hey @tomhughes ! So I removed the `first commit` as well as took out the `application.css` file you were right it did nothing. 

- Thank you.
